.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Run Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    env:
+      GOFLAGS: "-mod=vendor"
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    # if we don't do this, `gofmt` will want to rewrite all Go files due to bad line endings,
+    # because Git will convert all line endings to CRLF when cloning on windows
+    - name: Set Git to use Linux-style line endings
+      run: |
+        git config --global core.autocrlf false
+        git config --global core.eol lf
+    - name: Install Go
+      uses: actions/setup-go@v2-beta
+      with:
+        go-version: 1.14
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Debug Go env
+      run: go env
+    - name: Vet
+      run: make vet
+    - name: Website test
+      if: matrix.platform == 'ubuntu-latest'
+      run: make website-test GOPATH=`go env GOPATH`
```

Comment:
```suggestion
      shell: bash
      run: make -e GOPATH=$(go env GOPATH) website-test 
```

Reply:
Why? It seems to work fine as it is.

Reply:
- `bash` is safer to use as the shell might change
- `-e` is GNUMake option for env overrides (I am not sure why the hack with target `NAME=...` works)